back-end/Service.py

```diff
@@ -0,0 +1,280 @@
+        return redirect(request.url)
+    if file and allowed_file(file.filename):
+        file.save(os.path.join(app.config['UPLOAD_FOLDER'], request.form['name']))
+        resp = make_response(jsonify("Saved", 200))
+        return resp
+
+@app.route('/getImageFile/<fileName>', methods=['GET'])
+def get_image_file(fileName):
+    return send_from_directory(UPLOAD_FOLDER, fileName,as_attachment=False)
+
+
+
+@app.route('/login/<user>/<password>')
+def check_user(user, password):
+    db = DataAccess()
+    if len(user) == 0 or len(password) == 0:
+        return jsonify("Bad request"), 400
+    result = db.check_user(user, password)
+    if not result:
+        return jsonify("Authorization faled"), 401
+    elif result:
+        resp = make_response(jsonify(result["guid"]), 200)
+        resp.set_cookie('userID', result["guid"], 3600, secure = True, httponly=True)
+        return resp
+     
+
+# @app.route('/addImage', methods=['POST'])
+# def add_image():
+#     if 'admin' in request.cookies:
+#         data = json.loads(request.stream.read())
+#         db = DataAccessImage()
+#         image_handler = ImageHandler(db)
+#         image = Image(data["name"],data["imageType"],data["eventType"],data["location"])
+#         return jsonify(image_handler.add_image(image))
+#     else:
+#         return jsonify("Unauthorized"), 401
+
+@app.route('/addImage', methods=['POST'])
+def add_image():
+    data = json.loads(request.stream.read())
+    db = DataAccessImage()
+    image_handler = ImageHandler(db)
+    image = Image(data["name"],data["imageType"],data["eventType"],data["location"])
+    return jsonify(image_handler.add_image(image))
+    
+
+@app.route('/getAllImageTypes') 
+def get_all_image_types():
+    db = DataAccess()
+    return jsonify(db.get_all_image_types())
+
+
+@app.route('/getAllEventTypes') 
+def get_all_event_types():
+    db = DataAccess()
+    return jsonify(db.get_all_event_types())
+ 
+
+@app.route('/getAllImages') 
+def get_all_images():
+    db = DataAccessImage()
+    images_from_DB = db.get_all_images()
+    print(images_from_DB)
+    for image in images_from_DB:
+         image['path']=ConfigProvider.PHOTOS_BASE_URL+image['name']
+    resp = make_response(jsonify(images_from_DB, 200))
+    return resp
+
+@app.route('/updateImage', methods=['POST'])
+def update_image():
+    data = json.loads(request.stream.read())
+    db = DataAccessImage()
+    image_handler = ImageHandler(db)
+    image =  Image(data["name"], data["imageType"], data["eventType"], data["location"])
+    return jsonify(image_handler.edit_image(image))
+ 
+
+@app.route('/delete', methods=['POST'])
+def delete_image( ):
+    data = json.loads(request.stream.read())
+    db = DataAccessImage()
+    image_handler = ImageHandler(db)
+    image = Image(data["name"],data["imageType"],data["eventType"],data["location"])
+    return jsonify(image_handler.delete_image(image)) #RECHECK 
+
+
+@app.route('/addTimeUnit', methods=['POST'])

```

Comment:
your routes should not be named according to the method you are using.
please read on how you should structure rest APIs, ive sent you a link